pkg/edgeview/src/system.go

```diff
@@ -29,6 +30,20 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+const (
+	tarMaxBytes = 512000000 // max tar directory of 512 Mbytes data
+)
+
+var tarBlockDirs = []string{
+	"/persist/clear",
+	"/persist/vault",
```

Comment:
Can we also add `/run/domainmgr/cloudinit` here to avoid downloading of secrets from cloud-init files?

Reply:
good idea.